### PR TITLE
update location page metatags to use region

### PR DESCRIPTION
**Pull request by pnavarrc**

### CODE REVIEW

File: src/screens/LocationPage/utils.ts [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
-    ? `${countyName}, ${stateName} (${stateCode})`
-    : `${stateName} (${stateCode})`;
-  return locationName;
-}
-
-export function getPageTitle(projections: Projections): string {
-  const locationName = getLocationName(projections.fips);
-  return `${locationName} - COVID Risk Map & Key Metrics`;
+export function getPageTitle(region: Region): string {
+  return `${region.shortName} - COVID Risk Map & Key Metrics`;
```
Comment (ghop02) [inline src/screens/LocationPage/utils.ts:7]: Is short name the correct one to use? Feels like we should use fullName + the title will change a bit

Comment (chasulin) [inline src/screens/LocationPage/utils.ts:7]: look how efficient that is !

**Review by chasulin — APPROVED** [review]

File: src/screens/LocationPage/utils.ts (hunk 1 above)
Comment (chasulin) [reply src/screens/LocationPage/utils.ts:7]: ie.-

 Kimball County, Nebraska (NE) - COVID Risk Map & Key Metrics

Nebraska (NE) - COVID Risk Map & Key Metrics

Comment (pnavarrc) [reply src/screens/LocationPage/utils.ts:7]: right, not quite the same - I guess I would handle it type by type with a good fallback, good catch!

**Pull request merged**